RobotFramework_TestsuitesManagement/Config/CConfig.py

```diff
@@ -587,13 +588,13 @@ def __loadConfigFileLevel2(self) -> bool:
         if sTestCfgDir.startswith('.../'):
             sTestCfgDirStart = sTestCfgDir
             sTestCfgDir = sTestCfgDir[4:]
-            if os.path.exists(self.__sNormalizePath(os.path.abspath('./' + sTestCfgDir))):
+            if os.path.exists(CString.NormalizePath('./' + sTestCfgDir)):
                 sTestCfgDir = './' + sTestCfgDir
             else:
                 bFoundTestCfgDir = False
                 for i in range(0, 30):
```

Comment:
Magic number: 30

```
        for i in range(0, 30):
            sTestCfgDir = '../' + sTestCfgDir
``` 
This means: In case of the folder cannot be found, the loop is executed 30 times - even in case of there is no more parent folder available and you already reached the root. That's not nice.

Reaching the root should be a break criteria. Maybe you find some hints within the code of `CString::DetectParentPath` of the PythonExtensionsCollection. In `DetectParentPath` I implemented something similar.